components/dashboard/src/App.tsx

```diff
@@ -70,7 +70,7 @@ function App() {
         <div className="container">
             {renderMenu()}
             <Switch>
-                <Route path={["/", "/workspaces"]} exact render={
+                <Route path={["/", "/workspaces", "/login"]} exact render={
```

Comment:
Works ok but could we also update the URL path to `/` or `/workspaces`? Now both anonymous and logged in users now see the workspaces list (after logging in) but see the `/login` path in the URL. Feel free to merge and open a follow up issue fo this.

:M: :V: :C:

Reply:
Good point! I pushed again.